sem_covid/entrypoints/notebooks/rdf_transformer/rml_mapper_executor.py

```diff
@@ -0,0 +1,55 @@
+from abc import ABC, abstractmethod
+import json
+import requests as req
+from pathlib import Path
+from typing import List
+class RMLMapperABC(ABC):
+
+    @abstractmethod
+    def transform(self, rml_rule: str, sources: dict) -> str:
+        pass
+
+class RMLMapper(RMLMapperABC):
+
+    def __init__(self,
+                rml_mapper_url: str
+                ):
+        self.rml_mapper_url = rml_mapper_url
+    
+    def transform(self, rml_rule: str, sources: dict)-> str:
+        rml_mapper_query = {"rml": rml_rule, "sources": sources}
+        #print(rml_mapper_query)
+        rml_mapper_result = req.post(self.rml_mapper_url, json = rml_mapper_query)
+        if rml_mapper_result.ok:
+            return json.loads(rml_mapper_result.text)['output']
+        else:
+            print(rml_mapper_result)
+            return None
+
+
+def pack_data_sources(source_path: Path)->dict:
+    sources = { str(file_path.name) : str(file_path.read_text())
+                for file_path in list(source_path.glob("**/*"))}
+    return sources
+
+
+RML_MAPPER_URL = "http://srv.meaningfy.ws:4000/execute"
```

Comment:
1. constants in the middle of the file are never a good idea. Constants shall be placed either at the beginning or exported into some module __init__.py
2. We shall never hardcode the URL of the service. Expect it to be specified as an environment variable and picked up by the config.py module.